api/src/app.test.js

```diff
@@ -82,4 +95,74 @@ describe('a wish', () => {
     expect(wishList.length).toBe(1)
     expect(wishList[0].type).toBe(wishType)
   })
+
+  it('should return 422 and error message for nameValidation \'Child\'s name required\'', async () => {
+    const response = await request(app)
+      .post('/wishes')
+      .send({
+        child: {
+          name: '',
+          hometown: 'marietta',
+          illness: 'crecent',
+          age: '12'
+        },
+        type: 'go',
+        details: 'i want to be a real star',
+        sponsor: {
+          name: 'krabs',
+          logo: 'K',
+          links: []
+        }
+      })
+
+    expect(response.status).toBe(422)
+    expect(response.body.errors['child.name'].message).toBe('Child\'s name required')
+  })
+
+  it('should return 422 and error message for nameValidation \'Sponsor\'s name required\'', async () => {
+    const response = await request(app)
+      .post('/wishes')
+      .send({
+        child: {
+          name: 'patrick',
+          hometown: 'marietta',
+          illness: 'crecent',
+          age: '12'
+        },
+        type: 'go',
+        details: 'i want to be a real star',
+        sponsor: {
+          name: '',
+          logo: 'K',
+          links: []
+        }
+      })
+
+    expect(response.status).toBe(422)
+    expect(response.body.errors['sponsor.name'].message).toBe('Sponsor\'s name required')
+  })
+
+  it('should return 422 and error message for nameValidation \'Name must contain only letters of the alphabet\'', async () => {
```

Comment:
can you also through in a test to check again symbols too (for kid's name only, not sponsor)?

Reply:
Not sure what you mean. I have a written a test to check for symbols in kids name. I can remove the test for symbols in sponsors names if that what you mean?

Reply:
Hey @kidCaulfield, I believe he means throw in a test that ensure you can use diacritics in a child's name like `Matías` or `Nicolás`. If it works with those extra characters (just adding some to the happy path test should be sufficient) I can finalize the review and merge.